doc-surrealdb_versioned_docs/version-latest/querying/graphql/surrealist.mdx

```diff
@@ -0,0 +1,137 @@
+```
+
+:::note
+__Note:__ By running SurrealDB with the GraphQL module enabled, you are opting into an experimental feature. In SurrealDB GraphQL is still considered experimental and may not be as stable as querying via SurrealQL.  
+
+GraphQL is a pre-release feature with known security flaws. This is not recommended for production use. We welcome your feedback and contributions to help improve the feature and make it more robust.
+:::
+
+After starting the SurrealDB instance, you can navigate to the Surrealist to start a new connection. 
+
+### Start a new connection
+
+In the top left corner of the Surrealist, start a new connection. Ensure that the connection information is the same as the one you used to start the SurrealDB instance. In the example above we have set the user to `root` and the password to `root`. 
+
+:::note
+__Note:__ In a sandbox connection, querying via GraphQL is not supported. 
+:::
+
+Learn more about starting a connection in the [Surrealist documentation](/docs/surrealist/getting-started). 
+
+### Setting a namespace and database
+
+Before you can start writing queries, you need to set the [namespace](/docs/surrealdb/introduction/concepts/namespace) and [database](/docs/surrealdb/introduction/concepts/database) you want to use. For example you can set the namespace to `test` and the database to `test`. This will set the namespace and database for the current connection.
+
+Additionally, you can start [a serving in Surrealist](/docs/surrealist/concepts/local-database-serving) which also enables graphql automatically, this starts a server on `http://localhost:8000` by default for a root user with username and password `root`. 
+
+<ThemedImage
+  alt="Surrealist connection settings"
+  sources={{
+    light: useBaseUrl('img/surrealist-connection.png'),
+    dark: useBaseUrl('img/surrealist-connection.png'),
+  }}
+/>
+
+### Preparing your database
+
+Next, using the [SurrealQL query editor](/docs/surrealist/concepts/sending-queries) create some data. For example, create a new `user` table with fields for `firstName`, `lastName`, and `email` and add a new user to the database.
+
+```surql title="Creating a user table"
+
+DEFINE TABLE user SCHEMAFULL;
+
+-- Define some fields.
+DEFINE FIELD firstName ON TABLE user TYPE string;
+DEFINE FIELD lastName ON TABLE user TYPE string;
+DEFINE FIELD email ON TABLE user TYPE string
+  ASSERT string::is::email($value);
+DEFINE INDEX userEmailIndex ON TABLE user COLUMNS email UNIQUE;
+
+-- Create a new User 
+CREATE user CONTENT {
+    firstName: 'Jon',
+    lastName: 'Doe',
+    email: 'Jon.Doe@surrealdb.com',
+};
+```
+
+:::note
+
+__Note:__ You can also use GraphQL to query data from explicitly defined fields using the [`DEFINE FIELD` statement](/docs/surrealql/statements/define/field). Meaning that in schemaless mode, since the fields are not explicitly defined, you cannot query them using GraphQL.
```

Comment:
Recommend changing "Meaning that" to something like "As such, in" to avoid being a sentence fragment.